source/type/data.cpp

```diff
@@ -389,94 +389,6 @@ void Data::complete() {
     LOG4CPLUS_INFO(logger, "\t Building autocomplete " << autocomplete << "ms");
 }
 
-static ValidityPattern get_union_validity_pattern(const MetaVehicleJourney& meta_vj) {
-    ValidityPattern validity;
-    for (const auto& vj : meta_vj.get_base_vj()) {
-        if (validity.beginning_date.is_not_a_date()) {
-            validity.beginning_date = vj->base_validity_pattern()->beginning_date;
-        } else {
-            if (validity.beginning_date != vj->base_validity_pattern()->beginning_date) {
-                throw navitia::exception("the beginning date of the meta_vj are not all the same");
-            }
-        }
-        validity.days |= vj->base_validity_pattern()->days;
-    }
-    return validity;
-}
-
-void Data::build_associated_calendar() {
```

Comment:
Shouldn't you remove the commented line above, then (361) ?
Looks like it was ~moved~ copied into ed in this PR: https://github.com/CanalTP/navitia/pull/831/files#r23221968 :wink: